tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_fs_create_delete_loop.py

```diff
@@ -0,0 +1,239 @@
+            file_size = config.get("file_size", 1024)
+            threads = config.get("threads", 10)
+            client1.exec_command(
+                sudo=True,
+                cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create "
+                f"--threads {threads} --file-size {file_size} --files {files} --response-times Y --top "
+                f"{fuse_mount_dir}/fuse_ios",
+                long_running=True,
+            )
+
+            # Mounting Subvolume2 on Kernel
+            log.info(f"Mount subvolume {subvolume_name}_2 on Kernel Client")
+            kernel_mount_dir = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+            mon_node_ips = fs_util.get_mon_node_ips()
+            subvol_path_kernel, rc = client1.exec_command(
+                sudo=True,
+                cmd=f"ceph fs subvolume getpath {fs_name} {subvolume_name}_2 {subvolume_group_name}",
+            )
+            fs_util.kernel_mount(
+                [client1],
+                kernel_mount_dir,
+                ",".join(mon_node_ips),
+                sub_dir=f"{subvol_path_kernel.strip()}",
+                extra_params=f",fs={fs_name}",
+            )
+
+            client1.exec_command(
+                sudo=True, cmd=f"mkdir -p {kernel_mount_dir}/kernel_ios"
+            )
+            client1.exec_command(
+                sudo=True,
+                cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create "
+                f"--threads {threads} --file-size {file_size} --files {files} --response-times Y --top "
+                f"{kernel_mount_dir}/kernel_ios",
+                long_running=True,
+            )
+
+            # Mounting Subvolume3 on nfs
+            log.info(f"Mount subvolume {subvolume_name}_3 on NFS Client")
+            rhbuild = config.get("rhbuild")
+            nfs_servers = ceph_cluster.get_ceph_objects("nfs")
+            nfs_server = nfs_servers[0].node.hostname
+            nfs_name = "cephfs-nfs"
+
+            client1.exec_command(
+                sudo=True, cmd=f"ceph nfs cluster create {nfs_name} {nfs_server}"
+            )
+            if not wait_for_process(
+                client=client1, process_name=nfs_name, ispresent=True
+            ):
+                raise CommandFailed("Failed to create nfs cluster")
+            log.info("ceph nfs cluster created successfully")
+
+            nfs_export_name = "/export_" + "".join(
+                secrets.choice(string.digits) for i in range(3)
+            )
+
+            subvol_path_nfs, rc = client1.exec_command(
+                sudo=True,
+                cmd=f"ceph fs subvolume getpath {fs_name} {subvolume_name}_3 {subvolume_group_name}",
+            )
+            export_path = f"{subvol_path_nfs}"
+            nfs_mounting_dir = "/mnt/nfs_" + "".join(
+                secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+            )
+
+            if "5.0" in rhbuild:
```

Comment:
As 5.x is EOL and not adding test coverage to pacific suites, do you think we could remove this check here?

Reply:
Agreed. Made Changes